pkgs/desktops/gnome-3/extensions/unite-shell/default.nix

```
@@ -0,0 +1,39 @@
+{ stdenv, gnome3, fetchFromGitHub, xorg, glib, coreutils }:
+stdenv.mkDerivation rec {
+  pname = "gnome-shell-extension-unite-shell";
+  version = "44";
+
+  src = fetchFromGitHub {
+    owner = "hardpixel";
+    repo = "unite-shell";
+    rev = "v${version}";
+    sha256 = "0nqc1q2yz4xa3fdfx45w6da1wijmdwzhdrch0mqwblgbpjr4fs9g";
+  };
+
+  uuid = "unite@hardpixel.eu";
+
+  nativeBuildInputs = [ glib ];
+
+  buildInputs = [ xorg.xprop ];
+
+  buildPhase = ''
+    runHook preBuild
+    glib-compile-schemas --strict --targetdir=${uuid}/schemas/ ${uuid}/schemas
+    runHook postBuild
+  '';
+
+  installPhase = ''
+    runHook preInstall
+    mkdir -p $out/share/gnome-shell/extensions
+    cp -r ${uuid} $out/share/gnome-shell/extensions
+    runHook postInstall
+  '';
+
+  meta = with stdenv.lib; {
+    description = "Unite is a GNOME Shell extension which makes a few layout tweaks to the top panel and removes window decorations to make it look like Ubuntu Unity Shell.";
```

```suggestion
    description = "Unite is a GNOME Shell extension which makes a few layout tweaks to the top panel and removes window decorations to make it look like Ubuntu Unity Shell";
```
description should not have a period at the end.